src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -268,6 +268,13 @@
     agent-shutdown-fn :- IFn
     warn-legacy-auth-conf? :- schema/Bool
     metrics-service]
+   (when (get-in raw-config [:jruby-puppet :compat-version])
+     (log/errorf "%s %s %s"
```

Comment:
I know we're not doing i18n for logs yet, but to future-proof this should we wrap it in and i18n call?